compiler_opt/tools/combine_training_corpus.py

```diff
@@ -62,12 +67,20 @@ def main(argv):
       continue
 
     with tf.io.gfile.GFile(path, 'r') as f:
-      module_names.extend(
-          [os.path.join(sub_dir, name.rstrip('\n')) for name in f])
+      corpus_description = json.load(f)
+      module_names.extend([
+          os.path.join(sub_dir, name) for name in corpus_description['modules']
+      ])
+      del corpus_description['modules']
```

Comment:
nit: is there some other way that's not destructive to do the check? like just check there's no other key than 'modules' - non-mutating code is easier to reuse (so if we end up trying to reuse this, one less surprise)